visualisations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -0,0 +1,130 @@
+from webviz_plotly import Plotly
+import pandas as pd
+
+
+class FanChart(Plotly):
+    """Fan chart page element.
+
+    :param data: Either a file path to a `csv` file or a
+        :class:`pandas.DataFrame`. If a dataframe is given, each column is one
+        line in the chart. The dataframe index is used for the horizontal
+        values. Similarly for the `csv` file, where a special column named
+        ``index`` will be used for the horizontal values.
+    """
+    def __init__(self, data):
+        if isinstance(data, str):
+            self.data = pd.read_csv(data)
+            if 'index' in self.data.columns:
+                self.data.set_index(
+                    self.data['index'],
+                    inplace=True
+                )
+                del self.data['index']
+        else:
+            self.data = data
+
+        uniquelines = []
+        lines = []
+
+        for line_id in self.data['id']:
+            if line_id not in uniquelines:
+                uniquelines.append(line_id)
+
+        color = [
```

Comment:
It would be a good idea to use a predefined color palette, see for instance https://plot.ly/python/matplotlib-colorscales/.